client/tests/integration/unstable_network.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(clippy::restriction)]
+#![cfg(debug_assertions)]
```

Comment:
because there is some config parameter on Iroha that has `cfg(debug_assertions)`. And when I ran tests with `--release` it didn't compile. No, I don't think we ever run tests with `--release`